scoresheets/ObjectConfiguration.tex

```diff
@@ -2,23 +2,23 @@
 
 \begin{scorelist}
 	\scoreheading{Describing the objects} % 100
-	%\scoreitem[n]{50 \/ n}{Description touches upon each of the n objects}
-	\scoreitem[10]{5}{Description touches upon each of the n objects}
+	\scoreitem[10]{5}{Description touches upon each of the 10 objects}
```

Comment:
I find "Description touches upon" an unusual construct and its sense is quite ambiguous. Scoresheets must be straightforward. Of course if task changes, this needs also to change.